src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -146,15 +177,13 @@ var isInputEventSupported = false;
 if (ExecutionEnvironment.canUseDOM) {
   // IE9 claims to support the input event but fails to trigger it when
   // deleting text, so we ignore its input events.
-  // IE10+ fire input events to often, such when a placeholder
-  // changes or when an input with a placeholder is focused.
   isInputEventSupported = isEventSupported('input') && (
-    !('documentMode' in document) || document.documentMode > 11
+    !('documentMode' in document) || document.documentMode > 9
```

Comment:
reverts #4051 since this addresses that issue as well. Lets the polyfill be more focused since it has some downsides.